AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -207,11 +187,17 @@ - (void)setLayerBacked:(BOOL)layerBacked
 #pragma mark - Configuration
 @synthesize delegate = _delegate;
 
+- (void)setScrollEnabled:(BOOL)scrollEnabled{
```

Comment:
open bracket on next line - match the other methods.  Not a blocker, just FYI :)